modules/tinymce/src/themes/silver/main/ts/ui/dialog/UrlInput.ts

```diff
@@ -236,7 +236,7 @@ export const renderUrlInput = (
   const memUrlPickerButton = Memento.record(renderButton({
     name: spec.name,
     icon: Optional.some('browse'),
-    text: spec.label.getOr(''),
+    text: spec.picker_text.getOr(spec.label.getOr('')),
```

Comment:
Is there a followup for translating this?

Reply:
Do I need to create a follow-up ticket to add the string to the enterprise language pack?